mews-api.md

```diff
@@ -159,6 +155,11 @@ Returns configuration of the enterprise and the client.
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `includeUnsynchronizedRates` | `bool` | optional | If `true`, unsynchronized [`Rate plan`](mews-api.md#rate-plan)s will be returned as well. Unsynchronized rate plan means that Mews will not push prices and restrictions for that rate plan, but when a reservation comes with the rate plan code, Mews will link the correct rate plan with the reservation. |
+| `includeProducts` | `bool` | optional | If `true`, products mapped to a channel manager rate plan will be returned. Products mapped to a channel manager rate plan means that Mews sends a total price combined of night and product price in [`updatePrices`](https://mews-systems.gitbook.io/channel-manager-api/channel-manager-api#update-prices) request. |
```

Comment:
Does this also include products which have a CHM ID but are not attached to a rate?